std/experimental/ndslice/iteration.d

```diff
@@ -104,6 +104,8 @@ import std.meta;
 import std.experimental.ndslice.internal;
 import std.experimental.ndslice.slice; //: Slice;
 
+@trusted:
```

Comment:
What if something un-`@safe` is added in the future? It'll automatically be marked `@trusted`.